schema/annotation-source.yaml

```diff
@@ -237,7 +248,9 @@ $defs:
       target_proposition:
         extends: target_proposition
         $ref: "#/$defs/VariationGermlinePathogenicityProposition"
-
+      confidence_level:
+        extends: confidence_level
+        enum: [ "likely", "definitive" ]
```

Comment:
Here we define enum terms for the confidence_level field that would apply specifically to the VariationGermlinePathogenicityStatement class used to represent ClinVar data. The term "likely" obviously comes straight from ClinVar.  In previous talks  I think we had decided that "definitive" is the best term for when the classification is 'P' . . . but open to other ideas here.